### PR TITLE
Add FAQs

### PR DESCRIPTION
**Pull request by himanshugarg06**
https://github.com/bcnmy/docs/issues/206
https://github.com/bcnmy/docs/issues/174

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    
+you can pass the `index` in the config while creating the account as mentioned in the createSmartAccountClient [method](https://docs.biconomy.io/Account/methods#createsmartaccountclient). First smart account gets created with index 0 by default. 
+    
+**Where can I find more examples for SDK integration?**
+    
+https://github.com/bcnmy/sdk-examples
+    
+**How to increase the timeout in the waitForTransactionHash function**
+    
+Initialise the bundler with updated configuration, to increase the timeout (milliseconds).
```
Comment (NunoFilipeSantos) [inline docs/FAQ.md:22]:
```suggestion
**Will I get the same smart account address on all chains?**

Yes, you will get the same address on all chains as long as the owner's address is the same.
    
**How can I create another smart account using the same EOA on the same chain?**
    
you can pass the `index` in the config while creating the account as mentioned in the createSmartAccountClient [method](https://docs.biconomy.io/Account/methods#createsmartaccountclient). First smart account gets created with index 0 by default. 
    
**Where can I find more examples of SDK integration?**
    
https://github.com/bcnmy/sdk-examples
    
**How to increase the timeout in the waitForTransactionHash function**
    
Initialize the bundler with updated configuration, to increase the timeout (milliseconds).
```

File: docs/FAQ.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    
+**Which social login methods are supported?**
+    
+Explore services such as Web3Auth, Particle Network, Magic.link, or Lit Protocol to link a user's social identity with a cryptographic key. This key can then serve as the owner of the Smart Account. Be sure to check out various [signers](https://docs.biconomy.io/Account/signers/) integrations for this purpose.
+    
+**How do I enable session keys for users?**
+    
+Utilise the session key validation [module](https://docs.biconomy.io/Modules/sessionvalidationmodule) 
+    
+
```
Comment (NunoFilipeSantos) [inline docs/FAQ.md:44]:
```suggestion
**Which social login methods are supported?**
    
Explore services such as Web3Auth, Particle Network, Magic.link, or Lit Protocol to link a user's social identity with a cryptographic key. This key can then serve as the owner of the Smart Account. Be sure to check out various [signers](https://docs.biconomy.io/Account/signers/) integrations for this purpose.
    
**How do I enable session keys for users?**
    
Utilize the session key validation [module](https://docs.biconomy.io/Modules/sessionvalidationmodule) 
    
```

File: docs/FAQ.md (hunk 1 above)
Comment (NunoFilipeSantos) [reply docs/FAQ.md:22]: Please follow the same sentence pattern.

File: docs/FAQ.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+yes, bundler url is same for all testnets. For mainnet you can reach out to us.
+    
+**How to sign userOp manually?**
+    
+To sign the userOp, first calculate the userOpHash and then sign it using the same signer, account was created. you can find more details [here](https://hackmd.io/@xWzOEjWIRIKP22EKSdIgEg/Hk5SGcK-o).
+    
+**Can I just use Biconomy’s bundler and other smart account implementation?**
+    
+Yes, Biconomy's bundler seamlessly integrates with other smart account providers. As long as the user operation remains valid and complies with ERC 4337 specifications, integration is possible.
+
```
Comment (NunoFilipeSantos) [inline docs/FAQ.md:58]:
```suggestion
**Can I just use Biconomy’s bundler with another smart account implementation?**
    
Yes, Biconomy's bundler seamlessly integrates with other smart account providers. As long as the user operation remains valid and complies with ERC 4337 specifications, the integration is possible.

```

File: docs/FAQ.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+**Which ERC20 tokens are supported by the biconomy paymaster? How to add support for a custom ERC20 token?**
+    
+you can check out the list of supported networks under [paymaster](/Paymaster/supportedNetworks) and [bundler](/Bundler/supportedNetworks) section. you can reach out to us here for custom token requirement.
```
Comment (NunoFilipeSantos) [inline docs/FAQ.md:61]:
```suggestion
**Which ERC20 tokens are supported by the biconomy paymaster? How can I add support for a custom ERC20 token?**
    
You can check out the list of supported networks under the [paymaster](/Paymaster/supportedNetworks) and [bundler](/Bundler/supportedNetworks) sections. You can reach out to us here for custom token requirements.
```

File: docs/FAQ.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+    
+**How can I use any custom paymaster?**
+    
+It totally depends on how the `paymasterAndData` is parsed in the paymaster. Only restriction is that it should first encode the paymaster address after that encoding can be anything till the time decoding is consistent. Bundler does not care about the paymasterAndData.
+    
+
```
Comment (NunoFilipeSantos) [inline docs/FAQ.md:67]:
```suggestion
**How can I use any custom paymaster?**
    
It completely depends on how the `paymasterAndData` is parsed in the paymaster. The only restriction is that it should first encode the paymaster address after that encoding can be anything till the time decoding is consistent. Bundler does not care about the paymasterAndData.
    
```

File: docs/FAQ.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+**Can I fund the gasTank programmatically?**
+    
+You can programmatically deposit funds using the paymasterContract `depositFor` function by passing the `amount` and `paymasterId` which is the EOA address. For instance, you can invoke the following [function](https://polygonscan.com/address/0x00000f79b7faf42eebadba19acc07cd08af44789#writeContract#F3) on the Polygon chain.
+    
+**Are there APIs to create our own dashboard?**
+    
+yes, you can utilise the dashboard APIs to create a paymaster, change policies and even get the transactions data.
```
Comment (NunoFilipeSantos) [inline docs/FAQ.md:74]:
```suggestion
**Are there APIs to create my own dashboard?**
    
Yes, you can utilize the dashboard APIs to create a paymaster, change policies, and even get the transaction's data.
```

**Review by NunoFilipeSantos — CHANGES_REQUESTED** [review]
- Capitalization
- Sentence structure
- Gramamar checks

**Review by VGabriel45 — APPROVED** [review]

**Comment by VGabriel45** [comment]
Just 1 comment to add "Can I use other paymasters other than Biconomy ones with the biconomy sdk ?" re-phrase if you want, would be good to have

File: docs/FAQ.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+    
+To sign the userOp manually, first, calculate the userOpHash and then sign it using the same signer, the account is then created. you can find more details [here](https://hackmd.io/@xWzOEjWIRIKP22EKSdIgEg/Hk5SGcK-o).
+    
+**Can I just use Biconomy’s bundler with another smart account implementation?**
+    
+Yes, Biconomy's bundler seamlessly integrates with other smart account providers. As long as the user operation remains valid and complies with ERC 4337 specifications, the integration is possible.
+
+**Which ERC20 tokens are supported by the biconomy paymaster? How can I add support for a custom ERC20 token?**
+    
+You can check out the list of supported networks under the [paymaster](/Paymaster/supportedNetworks) and [bundler](/Bundler/supportedNetworks) sections. You can reach out to us here for custom token requirements.
```
Comment (joepegler) [inline docs/FAQ.md:60]: Could mention smartAccount.getSupportedTokens()  here

Comment (himanshugarg06) [reply docs/FAQ.md:60]: is this a new method in smartAccount?, yes we can mention here and add in methods as well.

Comment (himanshugarg06) [reply docs/FAQ.md:60]: @joepegler let me know once you add this method in the sdk, will add on this change in a new PR.

File: docs/FAQ.md [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+    
+**How do I enable session keys for users?**
+    
+Utilize the session key validation [module](https://docs.biconomy.io/Modules/sessionvalidationmodule) 
+    
+## Bundler & Paymaster
+
+**Is the bundler URL the same for all chains?**
+    
+Yes, the bundler URL is same for all testnets. For mainnet you can reach out to us.
```
Comment (joepegler) [inline docs/FAQ.md:48]: It's not exactly the same... The chainId changes. Could provide the example here?

**Review by arcticfloyd1984 — APPROVED** [review]

**Review by joepegler — CHANGES_REQUESTED** [review]

**Review by VGabriel45 — APPROVED** [review]

**Pull request merged**